client/js/components/statement/listStatements/ExportModal.vue

```diff
@@ -0,0 +1,200 @@
+<license>
+  (c) 2010-present DEMOS plan GmbH.
+
+  This file is part of the package demosplan,
+  for more information see the license file.
+
+  All rights reserved
+</license>
+
+<template>
+  <div>
+    <button
+      type="button"
+      @click.prevent="openModal"
+      class="btn--blank o-link--default inline-block"
+      data-cy="exportModal:open">
+      {{ Translator.trans('export.verb') }}
+    </button>
+
+    <dp-modal
+      ref="exportModalInner"
+      content-classes="w-11/12 sm:w-10/12 md:w-8/12 lg:w-6/12 xl:w-5/12">
+      <h2 class="mb-5">
+        {{ Translator.trans('export.statements') }}
+      </h2>
+
+      <section>
+        <h3 class="text-lg">
+          {{ Translator.trans('file.format') }}
+        </h3>
+        <div class="flex flex-row mb-5 mt-1">
+          <dp-radio
+            v-for="(exportType, key) in exportTypes"
+            :key="key"
+            :id="key"
+            class="mr-4"
+            :label="{
+              text: Translator.trans(exportType.label)
+            }"
+            :value="key"
+            :checked="isActive(key)"
+            @change="active = key" />
+        </div>
+      </section>
+
+      <section v-if="['docx', 'zip'].includes(active)">
+        <h3
+          id="docxColumnTitles"
+          class="inline-block text-lg mr-1">
+          {{ Translator.trans('docx.export.column.title') }}
+        </h3>
+        <dp-contextual-help
+          aria-labelledby="docxColumnTitles"
+          :text="Translator.trans('docx.export.column.title.hint')" />
+        <div class="grid grid-cols-5 gap-3 mt-1 mb-5">
+          <dp-input
+            v-for="(column, key) in docxColumns"
+            :key="key"
+            :id="key"
+            type="text"
+            :data-cy="column.defaultValue"
+            :placeholder="Translator.trans(column.placeholder)"
+            :width="column.width"
+            v-model="column.title" />
+        </div>
+      </section>
+
+      <div class="flex justify-end">
+        <dp-button-row
+          primary
+          secondary
+          :primary-text="Translator.trans('export.statements')"
+          :secondary-text="Translator.trans('abort')"
+          @primary-action="handleExport"
+          @secondary-action="handleAbort" />
+      </div>
+    </dp-modal>
+  </div>
+</template>
+
+<script>
+import {
+  DpButtonRow,
+  DpContextualHelp,
+  DpInput,
+  DpModal,
+  DpRadio,
+  sessionStorageMixin
+} from '@demos-europe/demosplan-ui'
+
+export default {
+  name: 'ExportModal',
+
+  components: {
+    DpButtonRow,
+    DpContextualHelp,
+    DpInput,
+    DpModal,
+    DpRadio
+  },
+
+  mixins: [sessionStorageMixin],
+
+  data () {
+    return {
+      active: 'docx',
+      docxColumns: {
+        left: {
+          width: 'col-span-1',
```

Comment:
ask Hans :) Here is a design template --> https://www.figma.com/proto/qKBgvF8gUAM1xIBA8Ubqgh/DPLAN-11773-Export-Modal?node-id=7-26&t=QGu9ukLPDXMD2ugA-1&scaling=min-zoom&content-scaling=fixed&page-id=4%3A5

It is probably because the column/title for Id needs less space than the other columns. As I see in the export document this column is smaller as well.